settings/DevHome.Settings/Views/FeedbackPage.xaml.cs

```diff
@@ -1,275 +1,302 @@
-        }
-
-        return osVersion.ToString();
-    }
-
-    private string GetPhysicalMemory()
-    {
-        CultureInfo cultures = new CultureInfo("en-US");
-
-        MEMORYSTATUSEX memStatus = default(MEMORYSTATUSEX);
-        memStatus.dwLength = (uint)Marshal.SizeOf(typeof(MEMORYSTATUSEX));
-        PInvoke.GlobalMemoryStatusEx(out memStatus);
-
-        var availMemKbToGb = Math.Round(memStatus.ullAvailPhys / ByteSizeGB, 2);
-        var totalMemKbToGb = Math.Round(memStatus.ullTotalPhys / ByteSizeGB, 2);
-
-        return "Physical Memory: " + totalMemKbToGb.ToString(cultures) + "GB (" + availMemKbToGb.ToString(cultures) + "GB free)";
-    }
-
-    private string GetProcessorArchitecture()
-    {
-        SYSTEM_INFO sysInfo;
-        PInvoke.GetSystemInfo(out sysInfo);
-        return "Processor Architecture: " + DetermineArchitecture((int)sysInfo.Anonymous.Anonymous.wProcessorArchitecture);
-    }
-
-    private string DetermineArchitecture(int value)
-    {
-        var arch = "Unknown architecture";
-        switch (value)
-        {
-            case 9:
-                arch = "x64";
-                break;
-            case 5:
-                arch = "ARM";
-                break;
-            case 12:
-                arch = "ARMx64";
-                break;
-            case 6:
-                arch = "Intel Itanium-based";
-                break;
-            case 0:
-                arch = "x86";
-                break;
-        }
-
-        return arch;
-    }
-
-    private string GetPlugins()
-    {
-        var pluginService = Application.Current.GetService<IPluginService>();
-        var plugins = pluginService.GetInstalledPluginsAsync(true).Result;
-        var pluginsStr = "Extensions: \n";
-        foreach (var plugin in plugins)
-        {
-            pluginsStr += plugin.PackageFullName + "\n";
-        }
-
-        return pluginsStr;
-    }
-
-    private async void BuildExtensionButtonClicked(object sender, RoutedEventArgs e)
-    {
-        await Launcher.LaunchUriAsync(new ("https://go.microsoft.com/fwlink/?linkid=2234795"));
-    }
-
-    private async void ReportSecurityButtonClicked(object sender, RoutedEventArgs e)
-    {
-        await Launcher.LaunchUriAsync(new ("https://github.com/microsoft/devhome/security/policy"));
-    }
-}
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Linq;
+using System.Management;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using System.Web;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Settings.Models;
+using DevHome.Settings.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Windows.System;
+using Windows.System.Profile;
+using Windows.Win32;
+using Windows.Win32.System.SystemInformation;
```

Comment:
There's a few using's being added here.  Are they all needed?

Reply:
They all seem to be needed, some were brought from the original FeedbackPage and others were added for Breadcrumb and more Settings oriented stuff.